### PR TITLE
Register events with images

### LINKED ISSUE #14: Register Events with Images
- Register an event with an Image
  - The user should be able to register an event, add images to it, and modify the images if desired
  - [ ] The user should be able to register an event
  - [ ] The user should be able to add images to the event
  - [ ] The user should be able to modify the images of the event

### PR DESCRIPTION
**Pull request by JoseLopezSolis**
**The user will now be able to:**
- Update **images**
- Edit **images**

**What do I?**
- It's now a new migration to the database called **active_storate** to let the user add an image to the "events" model.

image
image
image

co-authored-by: hugocruz1234 cruzhugo1997@gmail.com
co-authored-by: JoseLopezSolis jlopez114@ucol.mx
co-authored-by: ballo1988 rounderjards@gmail.com
co-authored-by: sr-meh hector.m.rm@hotmail.com
co-authored-by: jcardenas0 jcardenas0@ucol.mx

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,2 +1,2 @@
 class HomeController < ApplicationController
```
Comment (juperameza) [inline app/controllers/home_controller.rb:1]:
```suggestion
class HomeController < ApplicationController
       def index; end
```
You should have the index method

**Review by juperameza — CHANGES_REQUESTED** [review]
Just that one thing everything else looks good

**Review by juperameza — APPROVED** [review]
Lgtm

**Review by claunicole — APPROVED** [review]

**Comment by Ballo1988** [comment]
Done!

**Pull request merged**